shared/fetcher.ts

```diff
@@ -0,0 +1,65 @@
+import axios from "axios";
+import Cookies from "js-cookie";
+
+import getConfig from "next/config";
+import { ITokenEnum } from "./enum";
```

Comment:
please change relative import to alias import
e.g
import { BannerPage } from "component/Page/BannerPage"